cloudflared/config.yaml

```diff
@@ -23,8 +23,10 @@ schema:
       service: str
       disableChunkedEncoding: bool?
   tunnel_name: str?
-  tunnel_token: str?
-  post_quantum: bool?
   catch_all_service: str?
   nginx_proxy_manager: bool?
+  tunnel_token: str?
+  post_quantum: bool?
+  run_parameters:
+    - str?
```

Comment:
what do you think about restricting the possible options to the one we want to allow to be set? Something like:

```yaml
run_parameters:
    - match(^(--region|--retries|--protocol)=.*$)?
```

Reply:
Good idea, not sure though how to do it, since the value will be more then the parameter, e.g. "--region us" or "--protocol http2" or "--protocol quic" so we would need to specify all parameters plus there possible values, right?

Reply:
ok I just realized my mistake, thaty why we have the =.*$ Let me work on that and test it. Will do tomorrow.

Thanks for the review and the suggestion

Reply:
Should work now, ready for another review @elcajon